lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -342,13 +342,19 @@ def _section_e_commerce(course, access, paid_mode, coupons_enabled, reports_enab
 def _section_special_exams(course, access):
     """ Provide data for the corresponding dashboard section """
     course_key = six.text_type(course.id)
+    proctoring_provider = six.text_type(course.proctoring_provider)
```

Comment:
Instead of using library `six`, why not use the `str` function directly? Or do we even need to do string conversion? I thought the value of the `course.proctoring_provider` will be a string no matter what.